documentation/docs/90-additional-resources.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Additional Resources
+---
+
+### FAQs
+
+Please see the [SvelteKit FAQ](/faq) for solutions to common issues and helpful tips and tricks.
+
+The [Svelte FAQ](https://svelte.dev/faq) and [`vite-plugin-svelte` FAQ](https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md) may also be helpful for questions deriving from those libraries.
+
+### Integrations
+
+[Svelte Adders](https://sveltesociety.dev/templates#category-Svelte%20Add) allow you to setup many different integrations with a single command. Please see [sveltesociety.dev](https://sveltesociety.dev/) for a full listing of templates, components, and tools available for use with Svelte and SvelteKit. The SvelteKit FAQ also has a [section on integrations](faq#integrations), which may be helpful if you run into any issues.
```

Comment:
```suggestion
[Svelte Adders](https://sveltesociety.dev/templates#category-Svelte%20Add) allow you to setup many different integrations with a single command. Please see [sveltesociety.dev](https://sveltesociety.dev/) for a full listing of templates, components, and tools available for use with Svelte and SvelteKit. The SvelteKit FAQ also has a [section on integrations](/faq#integrations), which may be helpful if you run into any issues.
```

Maybe `/faq` (with a leading slash) just in case,